CMakeLists.txt

```diff
@@ -150,7 +150,7 @@ if (WITH_TESTS)
     enable_testing()
 endif()
 
-add_subdirectory(plugins)
+# add_subdirectory(plugins)
```

Comment:
@hobu @abellgithub  building plugins fail as it is unable to fine arbiter. Not sure why but it is cause i included the connector object tor the reader class